src/sknnr_spatial/datasets/_base.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+from importlib import resources
+
+import numpy as np
+import pandas as pd
+import rasterio
+import rioxarray
+import sknnr.datasets
+import xarray as xr
+from numpy.typing import NDArray
+
+DATA_MODULE = "sknnr_spatial.datasets.data"
+
+
+def _load_rasters_to_dataset(
+    file_names: list[str], *, var_names: list[str], module_name: str, chunks=None
+) -> xr.Dataset:
+    """Load a list of rasters from the data module as an xarray Dataset."""
+    das = []
+    for file_name, var_name in zip(file_names, var_names):
+        bin = resources.open_binary(module_name, file_name)
```

Comment:
I'm getting deprecation warnings on this line and line 35.
```
DeprecationWarning: open_binary is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    bin = resources.open_binary(module_name, file_name)
```
Is this expected behavior?

Reply:
On closer look, the compatibility port in `sknnr` was only needed for `3.8`, so I just switched to the new recommended approach for `>=3.9`.

Reply:
Interesting.  Looks like we're still supporting `3.8` in `sknnr` for compatibility with `scikit-learn`, so probably need to keep that compatibility port in that package until we deprecate `3.8`.